xknx/knxip/cemi_frame.py

```diff
@@ -87,6 +101,11 @@ def telegram(self, telegram: Telegram) -> None:
         self.dst_addr = telegram.destination_address
         self.payload = telegram.payload
 
+        print(
+            f"##### CEMIFrame-telegram {self.code}",
+            " ".join(f"{b:02x}" for b in self.payload.to_knx()),
+        )
```

Comment:
```suggestion
```

There should not be prints in the code. If there is anything viable to log there we can use a logger. If its only for development just do logger.debug